README.md

```diff
@@ -1,5 +1,5 @@
 # tallcraft-player-search
-Here's a [link](https://tallcraft-player-search.netlify.app/).
+Here's a [link](players.tallcraft.com).
```

Comment:
We don't have to link it here. There is already a link in the designated repository field.
What do you think about just removing it entirely?